xapian-applications/omega/omindexcheck.cc

```diff
@@ -1,5 +1,5 @@
 /** @file omindexcheck.cc
- * @brief test omindex
+ * @brief Auxiliar program of omindextest
```

Comment:
"Auxiliar" isn't an English word - I think you want "Auxiliary".

Reply:
Sorry, I am going to correct it.